### ISSUE TITLE
a11y: Color Contrast - Describe Resource

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13809]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF 
Ratio: 3.16:1
Main Content
- https://tools.ietf.org/html/rfc4648#section-4"

## Expected results

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Issue 172 resolved.
